docs/changelog/index.rst

```diff
@@ -1,6 +1,33 @@
 Changelog
 =========
 
+September 28, 2021
+-------------------
+
++-------------------------------+-------------------------------+
+| Module                        | Versions                      |
++===============================+===============================+
+| ``genie.webdriver``           | 21.9                          |
++-------------------------------+-------------------------------+
+
+August 31, 2021
+-------------------
+
++-------------------------------+-------------------------------+
+| Module                        | Versions                      |
++===============================+===============================+
+| ``genie.webdriver``           | 21.8                          |
++-------------------------------+-------------------------------+
+
+July 27, 2021
+-------------------
+
++-------------------------------+-------------------------------+
+| Module                        | Versions                      |
++===============================+===============================+
+| ``genie.webdriver``           | 21.7                          |
++-------------------------------+-------------------------------+
+
```

Comment:
it seems at some point in the past, we stop saying `- No change`. but better to have it.